base_partner_sequence/tests/test_base_partner_sequence.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
@lmignon oups. I was sure about that!
Sorry: I am quite sure I have seen some conversation about the topic and took it for granted.